### ISSUE TITLE
target  aarch64-linux-android Compilation error

### ISSUE THREAD

**Issue by androiddisk** [rust-lang/rust#123914]
`config.toml` modify
```
#target = build.host (list of triples)
```
After modification
```
target = ["x86_64-unknown-linux-gnu", "aarch64-linux-android"]
```
Compilation error
```
root@future:~/kooapk/rust/rust-1.77.2# python3 x.py build
Building bootstrap
    Finished dev [unoptimized] target(s) in 0.23s
WARNING: The `change-id` is missing in the `config.toml`. This means that you will not be able to track the major changes made to the bootstrap configurations.
NOTE: to silence this warning, add `change-id = 102579` at the top of `config.toml`
thread 'main' panicked at src/core/sanity.rs:59:13:


couldn't find required command: "aarch64-linux-android-clang"


note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Build completed unsuccessfully in 0:00:01
```

**Comment by onur-ozkan** [comment]
> couldn't find required command: "aarch64-linux-android-clang"

Do you have a toolchain for "aarch64-linux-android" target on your system?

**Comment by androiddisk** [comment]
> > couldn't find required command: "aarch64-linux-android-clang"
> 
> Do you have a toolchain for "aarch64-linux-android" target on your system?

```
[target.aarch64-linux-android]
android-ndk = "/root/rust/ndk/android-ndk"
```
have already been solved
